<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-REEGCHG" ignored="0" name="PerfectoReportiumExample" tests="1" failures="0" timestamp="23 Feb 2018 06:47:18 GMT" time="1.682" errors="0">
  <testcase name="@BeforeMethod beforeMethod" time="0.002" classname="org.mobile.mobileAssureFramework.ReportiumTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at org.mobile.mobileAssureFramework.ReportiumTest.beforeMethod(ReportiumTest.java:181)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod beforeMethod -->
  <testcase name="@AfterClass afterClass" time="0.001" classname="org.mobile.mobileAssureFramework.ReportiumTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at org.mobile.mobileAssureFramework.ReportiumTest.afterClass(ReportiumTest.java:252)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- @AfterClass afterClass -->
  <testcase name="sampleTestCase" time="0.0" classname="org.mobile.mobileAssureFramework.ReportiumTest">
    <skipped/>
  </testcase> <!-- sampleTestCase -->
</testsuite> <!-- PerfectoReportiumExample -->
